Source/Core/Core/IOS/Network/KD/NetKDRequest.cpp

```diff
@@ -194,14 +225,21 @@ NWC24::ErrorCode NetKDRequestDevice::KDDownload(const u16 entry_index,
 
   if (!response)
   {
-    ERROR_LOG_FMT(IOS_WC24, "Failed to request data at {}", url);
+    ERROR_LOG_FMT(IOS_WC24, "Failed to request data at {}. HTTP Status Code: {}", url,
+                  m_http.GetLastResponseCode());
```

Comment:
should this be extracted to:

```cpp
const auto last_response_code = m_http.GetLastResponseCode();
```

and then be used here and the other occurrences in the LogError call? Technically this path can do 4 curl API calls when only 1 seems to be needed.